openedx/features/edly/tests/test_utils.py

```diff
@@ -238,6 +240,47 @@ def test_user_cannot_login_on_requested_edly_organization(self):
 
         assert user_can_login_on_requested_edly_organization(self.request, self.user) is False
 
+    def test_user_can_login_on_requested_edly_organization_on_login_view(self):
+        """
+        Test user can login on the requested URL site if linked with its parent edly-organization.
+        """
+
+        user_password = 'password'
+        self.user.set_password(user_password)
+        self.user.save()
+
+        post_params = {'email': self.user.email, 'password': user_password}
+        request = RequestFactory().post('/', post_params)
+        request.site = SiteFactory()
+        request.session = self._get_stub_session()
+
+        edly_sub_organization_linked_to_site = self._create_edly_sub_organization()
+        edly_sub_organization_linked_to_user = EdlySubOrganizationFactory(
+            lms_site=request.site,
+            edly_organization=edly_sub_organization_linked_to_site.edly_organization,
+        )
+        self.edly_user_profile.edly_sub_organizations.add(edly_sub_organization_linked_to_user)  # pylint: disable=E1101
+
+        response = login_user(request)
+        assert response.status_code == 200
+
+    def test_user_cannot_login_on_requested_edly_organization_on_login_view(self):
+        """
+        Test user cannot login on the requested URL site if not linked with its parent edly-organization.
+        """
+
+        user_password = 'password'
```

Comment:
Since we are using this password for testing across multiple unit tests kindly declare it as constant.

Reply:
done